modules/extension/app-schema/app-schema/src/main/java/org/geotools/appschema/jdbc/JoiningJDBCFeatureSource.java

```diff
@@ -361,6 +365,14 @@ protected static String createAlias(String typeName, Set<String> tableNames) {
         return alias;
     }
 
+    public String selectSQL(
```

Comment:
Add a comment, could be the same as the former method.

Reply:
fixed